Makefile

```diff
@@ -542,6 +542,7 @@ changelog: check_version_coherence
 release: check_version_coherence check_apidocs
 	@PROJECT_VER=($$(poetry version)) && \
 	PROJECT_VER="$${PROJECT_VER[1]}" && \
+	./script/release_utils/check_branch_name.sh PROJECT_VER && \
```

Comment:
Isn't it missing dollars on PROJECT_VER?

Reply:
ah yes you're right of course